control_scheduling:
  ros__parameters:
    initial_ctrl_dof:
      x: 400.0
      y: 70.0
      z: 20.0
      yaw: 0.0
      pitch: 0.0
      roll: 0.0
      arm_yaw: 10.0
      arm_pitch: 10.0